subprojects/ci/k8s-deployments-cleaner/src/main/kotlin/com.avito.ci/ClearK8SDeploymentsByNamespaces.kt

```diff
@@ -37,7 +37,7 @@ class ClearK8SDeploymentsByNamespaces(
                         deployments.checkDeploymentLeak(deployment)
                     }
             } catch (e: Throwable) {
-                throw RuntimeException("Error when checked leak in namespace=$namespace", e)
+                throw RuntimeException("Error when checked leak in namespace=$namespace. If reason wasn't clear read previous discussion http://links.k.avito.ru/Az", e)
```

Comment:
Can we extract any results from this conversation and add them to the error?

Reply:
Everything has been extracted already. Look inside of `checkDeploymentLeak`